app/services/force/lease_service.rb

```diff
@@ -0,0 +1,48 @@
+module Force
+  # Encapsulate all Salesforce Lease__c querying and updating functions.
+  class LeaseService < Force::Base
+    def lease(application_id)
+      builder.from(:Lease__c)
+             .select(:Id,
+                     :Unit__c,
+                     :Lease_Start_Date__c,
+                     :Lease_Status__c,
+                     :Monthly_Parking_Rent__c,
+                     :Total_Monthly_Rent_without_Parking__c,
+                     :Monthly_Tenant_Contribution__c)
+             .where_eq(:Application__c, application_id, :string)
+             .transform_results { |results| massage(results) }
+             .query
+             .records
+             .first
+    end
+
+    def submit_lease(lease, application_id, _primary_contact_id)
```

Comment:
I feel this method might be a bit too long,  what about having a 

`create_lease` and `update_lease`

and then in `submit_lease`

```
if lease[:id]
   update_lease(lease)
else
  create_lease(lease)
end
```